### PR TITLE
🍾💻🚗 Add Lucy's internship post (closes #23 🏀)

### LINKED ISSUE #23: Blog post about internship + googledrive
@LucyMcGowan Let's develop this to a certain point elsewhere, but this will eventually give you the satisfaction of closing an issue when it's done!

### PR DESCRIPTION
**Pull request by LucyMcGowan**
**TODO**
- [x] needs a better title 🙈 
- [x] what category should it be?
- [x] Should I crop my face out of the commits? I feel like there is way too much of my face, but without it they don't quite look like commits anymore 🤔. 

@jennybc & @hadley do you mind taking a :eyes: - I don't think I can request a review (maybe because I don't have write access?) 

😱 I JUST NOTICED that the issue this closes is #23, which is Michael Jordan's number, which ties in to dribbling 🏀 + space jam. Ergo, EVERYTHING IS META AND [AWESOME](https://m.youtube.com/watch?v=StTqXEQ2l-Y) 🎉

### CODE REVIEW

File: content/articles/2017-08-25-lucy-experience.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+title: "Lucy's Internship Experience"
```
Comment (hadley) [inline content/articles/2017-08-25-lucy-experience.md]: Needs emoji in title?

File: content/articles/2017-08-25-lucy-experience.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+photo:
+  url: https://twitter.com/_inundata
+  author: Karthik Ram
+categories: [Other]
+---
+
+
+_Over the past several months I have been working with Jenny Bryan on the googledrive package. I wanted to share a bit about the process and some things I learned along the way. While brainstorming this post, I went back through my GitHub commits to remind myself of the journey - it turns out they do a rather good job of showing the scope of this!_
+
+## _Day to day_ <br> <img src = "/images/lucy-internship/01_first.png" width = "400pt"> </img>
```
Comment (hadley) [inline content/articles/2017-08-25-lucy-experience.md]: `width = "400"`

File: content/articles/2017-08-25-lucy-experience.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+
+## _Absorbing all things "tidyverse"_ <br> <img src = "/images/lucy-internship/03_markdown-docs.png" width = "400pt"></img>
+
+<center>
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">🌞 musing: learning a new coding style is like putting on freshly washed jeans; a struggle, but once I get it I&#39;m like that looks GOOD 💃 <a href="https://t.co/5QcEwEcI23">pic.twitter.com/5QcEwEcI23</a></p>&mdash; Lucy 🌻 (@LucyStats) <a href="https://twitter.com/LucyStats/status/873587893754843136">June 10, 2017</a></blockquote> <script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+</center>
+
+The tidyverse has an [R Style Guide](http://style.tidyverse.org). My first pass at incorporating the style was switching my roxygen documentation to [markdown](https://github.com/klutometis/roxygen/blob/master/vignettes/markdown.Rmd) (this can be enabled by adding `Roxygen: list(markdown = TRUE)` to the `DESCRIPTION` file). This is an example of something I didn't even know existed prior to this project. Other pieces I absorbed include:
```
Comment (hadley) [inline content/articles/2017-08-25-lucy-experience.md]: Mention roxygen2md?

Comment (LucyMcGowan) [reply content/articles/2017-08-25-lucy-experience.md]: 😱 🙊 I didn't even know about that! I'll definitely add it.

**Comment by jennybc** [comment]
I think it looks great too!

I think the title is fine. And do not have a better idea at the moment.

I hope @hadley will weigh in with category suggestion. It looks like an equally good (or bad) fit for "learn", "package", "programming", or "other" to me, which is not very helpful.

I'd leave the commit screenshots as they are. I didn't even notice the "repeated face" thing until I finished reading the post and read this thread. I don't think it's an issue.

**Comment by hadley** [comment]
Maybe learn? Once we have more posts we'll probably go through and recategorise everything.

**Comment by jennybc** [comment]
I just noticed that @karawoo's post is "other", so these two posts should be categorized consistently.

**Comment by karawoo** [comment]
I can change mine to learn

**Comment by hadley** [comment]
Lets plan to publish both on Friday - can you please set the date accordingly?

**Comment by hadley** [comment]
Thanks!

**Pull request merged**